thirdeye/thirdeye-frontend/app/pods/self-serve/create-alert/controller.js

```diff
@@ -346,7 +346,9 @@ export default Ember.Controller.extend({
   callReplayStart(functionId, startTime, endTime) {
     const granularity = this.get('graphConfig.granularity').toLowerCase();
     const speedUp = granularity.includes('hour') || granularity.includes('day');
```

Comment:
Actually if it's hour/day speedup should be "false". Minutely should have speedup to be "true" since it has more data.